.gitignore

```diff
@@ -1 +1,3 @@
 *.pyc
+.gitignore
```

Comment:
Please do not do this! `.gitignore` should never be ignored. `.DS_Store` should be ignored in your global .gitignore if you use macOS using a global `core.excludesfile` setting, rather than adding the ignore to each repository you touch.

Reply:
Yeah I was compounding stupid mistakes here ;)